views/export/index.tt

```diff
@@ -0,0 +1,106 @@
+[%# export/index.tt
+
+Journal export functionality
+
+Authors:
+    Mark Smith <mark@dreamwidth.org>
+
+Copyright (c) 2015 by Dreamwidth Studios, LLC.
+
+This program is free software; you may redistribute it and/or modify it under
+the same terms as Perl itself.  For a copy of the license, please reference
+'perldoc perlartistic' or 'perldoc perlgpl'.
+
+%][%- sections.title = title_ml | ml -%]
+
+[%- CALL dw.ml_scope( '/export.bml' ) -%]
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+[% sections.title = '.title' | ml %]
+
+[%- authas_form -%]
+
+<p>[% '.description' | ml %]</p>
+
+<form action='export_do?authas=[% u.username %]' method='post'>
+
+<fieldset>
+
+<div class="row">
```

Comment:
If you wanted to restrict the width of the entire form, might be easier to just do:

    <fieldset class="row"><div class="columns large-x"></div></fieldset>

Use the offset / end classes from http://foundation.zurb.com/docs/components/grid.html too as appropriate to remove the need for the divs.